imageroot/systemd/user/nginx.service

```
@@ -0,0 +1,26 @@
+[Unit]
+Description=Podman  nginx.service
+BindsTo=webserver.service
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=%S/state/environment
+WorkingDirectory=%S/state
+Restart=always
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/nginx.pid %t/nginx.ctr-id
+ExecStartPre=/usr/local/bin/runagent expand-vhosts
+ExecStart=/usr/bin/podman run --conmon-pidfile %t/nginx.pid \
+    --cidfile %t/nginx.ctr-id --cgroups=no-conmon \
+    --pod-id-file %t/webserver.pod-id --replace -d --name  nginx \
+    --volume ./conf.d:/etc/nginx/conf.d:z --volume websites:/usr/share/nginx/html:Z \
+    docker.io/nginx:1.20.2
```

The Nginx image is a real module dependency: let's use the env variable provided by the core through the `org.nethserver.images` image label.

```suggestion
    ${NGINX_IMAGE}
```